### PR TITLE
added option to remove RHS also when removing unused variables

### LINKED ISSUE #91: Option to drop RHS when removing unused variables
`autoflake` correctly keeps the RHS when removing unused variables, e.g.
```
unused_var = some_operation()
```
becomes
```
some_operation()
```

This makes sense as `some_operation()` may change something elsewhere, so is not removed.

One problem however, is that now `some_operation()` is simply a statement, so there's a stronger assumption that it's there for a reason. The moment that the unused variable is removed seems the optimal time for devs to decide whether the RHS should be kept.

In my case, I'd like to remove many of these such lines, in a flow such as:
```
First we run autoflake with this new flag enabled, such that the RHS is dropped
$ autoflake --in-place --remove-unused-variables --drop-unused-variable-rhs $(find . -name "*.py)

Next use git's -p option to filter which of those removed statements 
$ git add -p
< only stage changes whose RHS statements we don't want to keep >

Next we drop all the non-staged changes (the ones which we wanted to keep the RHS)
$ git checkout .

Now re-run autoflake but don't remove RHS
$ autoflake --in-place --remove-unused-variables $(find . -name "*.py)

And stage the removed unused variables (where RHS is kept)
$ git add -p

Commit and profit
$ git commit -m "removed all the garbage"
```

I think this would be a useful addition. However `autoflake` doesn't have many options, so not sure if this would be considered too rare of a use case to add?

I'll probably code this regardless to solve my case, the main question is whether or not to spend the time to just hack it together, or do it properly and PR in.

### PR DESCRIPTION
**Pull request by das-intensity**
close #91

### CODE REVIEW

**Comment by fsouza** [comment]
@das-intensity thanks for contributing! Can you resolve the conflicts?

**Comment by das-intensity** [comment]
@fsouza Done :) I also tried to match the formatting a little better, but am unclear on some of the stylings so let me know if you'd prefer some different whitespace config somewhere.

**Comment by fsouza** [comment]
> @fsouza Done :) I also tried to match the formatting a little better, but am unclear on some of the stylings so let me know if you'd prefer some different whitespace config somewhere.

For style, if `pre-commit` is happy I'm happy haha If the build fails you can run `pre-commit run -a`

**Comment by das-intensity** [comment]
Ahh thanks, I hadn't used pre-commit before. All issues fixed there. Please cancel run 71 but 72 should be good to go.

File: autoflake.py [hunk 1]
```diff
@@ -1192,6 +1206,11 @@ def _main(argv, standard_out, standard_error, standard_input=None):
         '--remove-unused-variables', action='store_true',
         help='remove unused variables',
     )
+    parser.add_argument(
+        '--remove-rhs-for-unused-variables', action='store_true',
```
Comment (fsouza) [inline autoflake.py:1210]: Can you also add `remove-rhs-for-unused-variables` to the config function? (The validation in the function `merge_configuration_file`)

**Review by fsouza — COMMENTED** [review]
Thanks for contributing! Just one thing and I think it should be good to go.

File: autoflake.py (hunk 1 above)
Comment (das-intensity) [reply autoflake.py:1211]: done

**Review by fsouza — APPROVED** [review]

**Pull request merged**